plenum/test/helper.py

```diff
@@ -1271,6 +1274,14 @@ def create_commit_with_bls_sig(req_key, bls_sig):
     return Commit(*params)
 
 
+def create_commit_with_bls_sigs(req_key, bls_sig):
```

Comment:
I think we need to include `bls_multi_sigs` into all test helpers for creation of Commit. If it's not specified, a defualt can be added (like `[bls_sig]`).